openmetadata-service/src/main/resources/json/data/testConnections/pipeline/stitch.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Stitch",
+  "displayName": "Stitch Test Connection",
+  "description": "This Test Connection validates the access against the server and basic metadata extraction of pipelines.",
+  "steps": [
+      {
+          "name": "CheckAccess",
+          "description":  "Check if the instance is reachable to fetch the pipeline details.",
```

Comment:
shouldn't we at least validate if we can get some metadata out?

Reply:
in the checkaccess we're testing api with getting pipelines data. ya but splitting the two checks one for access and one for pipeline/metadata will be better idea.